### PR TITLE
Eva/tgsrv 148 layerservice configreader

### PR DESCRIPTION
**Pull request by gevos**

### CODE REVIEW

File: src/starterConfig.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      data: {
+        type: 'wmts',
+        url:
+          `${CONF.tilesUrl}/wmts/netzkarte_relief_grey_webmercator/` +
+          'webmercator/{TileMatrix}/{TileCol}/{TileRow}.png',
+        matrixSet: 'webmercator',
+        requestEncoding: 'REST',
+        projectionExtent: PROJ.extent,
+        resolutions: [
+          156543.033928,
```
Comment (oterral) [inline src/starterConfig.js]: You could use the wmts Resolutions from src/utils/WMTSTileGrid3857.js

**Review by oterral — CHANGES_REQUESTED** [review]
I would keep src/utils/WMTSTileGrid3857.js and src/utils/WMTSTileGrid2056.js files, it always useful to get the list of resolutions for a projection. Since we never know when a project is webmercator or swiss.

We could also have 2 starterConfig one in swiss projection and one in mercator so it' easier to choose.

File: src/starterConfig.js (hunk 1 above)
Comment (gevos) [reply src/starterConfig.js]: If I remember that correctly, @mhaertwig suggested to integrate the WMTSTileGrid configuration into the new config where also the layers are defined. Is that correct? What was the reason?
I have just integrated these two WMTSTileGrid files into the new layer config file. So, the information does not get lost.

**Comment by gevos** [comment]
> I would keep src/utils/WMTSTileGrid3857.js and src/utils/WMTSTileGrid2056.js files, it always useful to get the list of resolutions for a projection. Since we never know when a project is webmercator or swiss.

I have re-created the WMTSTileGrid files and unified the naming.

> We could also have 2 starterConfig one in swiss projection and one in mercator so it' easier to choose.

As soon as we need that, we could implement it.

**Review by oterral — APPROVED** [review]
thx very good

File: src/starterConfig.js (hunk 1 above)
Comment (oterral) [reply src/starterConfig.js]: I don't know what the reason were.

**Pull request merged**